From ac3e7d316efee0b7a2b953aebdd3268f2b7c31c5 Mon Sep 17 00:00:00 2001
From: Thomas D.
Date: Thu, 13 Aug 2015 17:50:36 +0200
Fix: Parallel build issue "cannot find ../runtime/.libs/librsyslog.a: No such file or directory" (Fixes #479)

This issue was introduced with commit fb4fd2ddd2f08380ad65a8cafc5f124890b136ad.
---
 runtime/Makefile.am | 1 +
 1 files changed, 1 insertions(+)

diff --git a/runtime/Makefile.am b/runtime/Makefile.am
index 1f3e350..d02ed4f 100644
--- a/runtime/Makefile.am
+++ b/runtime/Makefile.am
@@ -186,6 +186,7 @@ if ENABLE_LIBGCRYPT
    libgcry_la_SOURCES = libgcry.c libgcry_common.c libgcry.h
    libgcry_la_CPPFLAGS = $(RSRT_CFLAGS) $(LIBGCRYPT_CFLAGS)
    pkglib_LTLIBRARIES += lmcry_gcry.la
+   lmcry_gcry_la_DEPENDENCIES = librsyslog.la
    lmcry_gcry_la_SOURCES = lmcry_gcry.c lmcry_gcry.h
    lmcry_gcry_la_CPPFLAGS = $(RSRT_CFLAGS) $(LIBGCRYPT_CFLAGS)
    lmcry_gcry_la_LDFLAGS = -module -avoid-version \
-- 
2.5.0

